integration_tests/pages/overview/EducationAndTrainingPage.ts

```diff
@@ -40,13 +40,18 @@ export default class EducationAndTrainingPage extends Page {
     return this
   }
 
-  hasCompletedInPrisonQualificationsDisplayed(): EducationAndTrainingPage {
-    this.completedInPrisonQualificationsTable().should('be.visible')
+  hasCompletedInPrisonQualificationsLast12MonthsDisplayed(): EducationAndTrainingPage {
+    this.completedInPrisonQualificationsLast12MonthsTable().should('be.visible')
     return this
   }
 
-  doesNotCompletedInPrisonQualificationsDisplayed(): EducationAndTrainingPage {
-    this.completedInPrisonQualificationsTable().should('not.exist')
+  hasCompletedInPrisonQualificationRecordDisplayed(expected: string): EducationAndTrainingPage {
```

Comment:
So this is just asserting the table is visible .. 🤔 
I wonder if that is part of the problem in this PR .... to be able to assert there are completed courses displayed, we need to assert the table has some records (`<tr>`s perhaps?), and/or does also contain the "no records" message